packages/dds/tree/docs/main/compatibility.md

```diff
@@ -234,6 +234,16 @@ To make a breaking change in `SchemaChangeCodec`,
     -   Make this write version create edit manager & message codecs of the appropriate versions
     -   Be sure to document code saturation requirements which must be met before the new version can be used
 
+## Example
+
+A new format was introduced for optional-field in [this PR](https://github.com/microsoft/FluidFramework/pull/20341).
+	- This PR should have also included code changes which began writing this format in messages! That was instead made shortly after in [this commit](https://github.com/microsoft/FluidFramework/commit/0fafbebcd3324fc481bd8464f09ab15d595b4a57).
+That format was added as an option for `SharedTree` users in [this PR](https://github.com/microsoft/FluidFramework/pull/20615).
+
+Future changes introducing a new format can do so in a single PR if desired.
```

Comment:
This seems in tension with the next line(s). I think it would be preferable to have the docs just prescribe "expose the new format in `SharedTreeFormatVersion` in a separate PR" (With the explanation of the benefits). If the reader can reason about why this doesn't matter for their case then they can make the leap to "I can just do it all in one PR" on their own.

Reply:
Fair point. I think just removing this line is sufficient?
```suggestion
```